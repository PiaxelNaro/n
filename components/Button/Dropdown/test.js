// @flow
import React from 'react';
import { shallow } from 'enzyme';
import DropdownButton from '.';

describe('<DropdownButton>', () => {
    it('renders without label', () => {
        expect(shallow(<DropdownButton value="Country" />)).toMatchSnapshot();
    });

    it('renders with label', () => {
        expect(
            shallow(<DropdownButton value="Ad Mob" label="Network" labelIcon={<i />} />)
        ).toMatchSnapshot();
    });
});
