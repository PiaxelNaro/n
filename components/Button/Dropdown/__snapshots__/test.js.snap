// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DropdownButton> renders with label 1`] = `
<div
  style={
    Object {
      "width": "110px",
    }
  }
>
  <Dropdown__DropdownLabel
    gutter="3px"
  >
    <i />
    <Text__Text
      ellipsis={true}
    >
      Network
    </Text__Text>
  </Dropdown__DropdownLabel>
  <Dropdown__StyledDropdownButton>
    <FlexGrid>
      <FlexGrid.Item
        flex={1}
      >
        <Text__Text
          ellipsis={true}
        >
          Ad Mob
        </Text__Text>
      </FlexGrid.Item>
      <MdArrowDropDown
        size={13}
      />
    </FlexGrid>
  </Dropdown__StyledDropdownButton>
</div>
`;

exports[`<DropdownButton> renders without label 1`] = `
<div
  style={
    Object {
      "width": "110px",
    }
  }
>
  <Dropdown__StyledDropdownButton>
    <FlexGrid>
      <FlexGrid.Item
        flex={1}
      >
        <Text__Text
          ellipsis={true}
        >
          Country
        </Text__Text>
      </FlexGrid.Item>
      <MdArrowDropDown
        size={13}
      />
    </FlexGrid>
  </Dropdown__StyledDropdownButton>
</div>
`;
